.github/include/ci.py

```diff
@@ -256,6 +254,57 @@ def run_runtime_tests():
     )
 
 
+def fuzz():
+    """
+    Run a basic fuzz smoke test.
+    """
+    # Setup the core pattern.
+    shell(cmd=["sh", "-c", "echo core | tee /proc/sys/kernel/core_pattern"], as_root=True)
```

Comment:
Can we get away without this? It'd be nice to not modify host state too much, as `ci.py` is intended to be run locally as well as in CI. Given I don't see any slurping of the cores (so they're lost anyways), maybe just set `AFL_I_DONT_CARE_ABOUT_MISSING_CRASHES`?

Reply:
LOL, apparently you should only [use that](https://aflplus.plus/docs/env_variables/) if you are Jakub.

But this sounds fine, I will refresh and plumb `AFL_I_DONT_CARE_ABOUT_MISSING_CRASHES` with a comment explaining it.

Reply:
Done, I'm just going to verify that CI passes before resolving.

Reply:
Seems to be working.